website/project/decorators.py

```diff
@@ -98,7 +122,7 @@ def check_key_expired(key, node, url):
 
     return url
 
-
+# FIXME(sloria): How often is this called with @must_be_valid_project? -- both have similar beginnings
```

Comment:
Haha, this was something I meant to bring up to you a long, long time ago but is no longer relevant. I'll remove that